<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculateServiceCostDialog</class>
 <widget class="QDialog" name="CalculateServiceCostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate Service Cost</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Small Icons/Icons/Prigram_Icon.png</normaloff>:/Small Icons/Icons/Prigram_Icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addItemButton">
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeSelectedItemButton">
       <property name="text">
        <string>Remove Selected Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="itemsTableWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Life Interval (min)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>CalculateServiceCostDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
